### DISCUSSION TITLE
Is it possible to define additional tokens beside Dogecoin?

### DISCUSSION THREAD

**Discussion by tomerarm** [dogecoin/dogecoin#2875] [Q&A]
Hi
Perhaps a noob question, but is it possible to define additional tokens beside Dogecoin on the same Dogecoin network?
Thank you in advance

**Comment by patricklodder** [comment]
No, Dogecoin does not support this intrinsically, it is really a chain dedicated to managing the DOGE currency. There are second-layer consensus algorithms available (e.g. Dogeparty) but those can be easily censored, because the transactions have no meaning inside the Dogecoin consensus layer itself.

What type of use case did you have in mind for your token? There are plenty of solutions out there that you can use, just got to pick the right one for your use case.
